packages/subgraph/schema.graphql

```diff
@@ -702,6 +760,7 @@ type BurnedEvent implements Event @entity {
     order: BigInt!
     operator: Bytes!
     from: Bytes!
+    token: Bytes!
```

Comment:
I didn't see this mapped in: `packages/sdk-core/src/mapGetAllEventsQueryEvents.ts`